### PR TITLE
Merge back stable to master after RC66 release

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

File: libraries/render-utils/src/AntialiasingEffect.h [hunk 1]
```diff
@@ -26,7 +26,11 @@ class JitterSampleConfig : public render::Job::Config {
         Q_PROPERTY(bool stop MEMBER stop NOTIFY dirty)
         Q_PROPERTY(int index READ getIndex NOTIFY dirty)
 public:
-    JitterSampleConfig() : render::Job::Config(true) {}
+    JitterSampleConfig() : render::Job::Config(true) {
```
Comment (samcake) [inline libraries/render-utils/src/AntialiasingEffect.h]: Please, do not port back any of the changes to this file "AntialisingEffect.h"

File: scripts/developer/utilities/render/antialiasing.qml [hunk 2]
```diff
@@ -36,7 +36,7 @@ Rectangle {
             Row {
                 spacing: 10
                 id: fxaaOnOff
-                property bool debugFXAA: false
+                property bool debugFXAA: true
```
Comment (samcake) [inline scripts/developer/utilities/render/antialiasing.qml]: Please, do not port back any of the changes to this file "Antialising.qml"

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8879/app_12927_PR_758bf09b302e6db6e67d89f6887087a1273dbf71.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8879/HighFidelity-Beta-PR12927-758bf09b302e6db6e67d89f6887087a1273dbf71.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8879/HighFidelity-Beta-Interface-PR12927-758bf09b302e6db6e67d89f6887087a1273dbf71.exe

**Pull request closed without merging**